diff --git a/node_modules/strapi-plugin-icon-picker/README.md b/node_modules/strapi-plugin-icon-picker/README.md
old mode 100644
new mode 100755
diff --git a/node_modules/strapi-plugin-icon-picker/admin/src/components/Initializer/index.js b/node_modules/strapi-plugin-icon-picker/admin/src/components/Initializer/index.js
old mode 100644
new mode 100755
diff --git a/node_modules/strapi-plugin-icon-picker/admin/src/components/Input/index.js b/node_modules/strapi-plugin-icon-picker/admin/src/components/Input/index.js
old mode 100644
new mode 100755
index 558aae0..f3c41c3
--- a/node_modules/strapi-plugin-icon-picker/admin/src/components/Input/index.js
+++ b/node_modules/strapi-plugin-icon-picker/admin/src/components/Input/index.js
@@ -1,6 +1,7 @@
-import React from "react";
+import React, { useState, useRef } from 'react';
 import { Grid, GridItem } from "@strapi/design-system/Grid";
 import { fab } from "@fortawesome/free-brands-svg-icons";
+import { fas } from "@fortawesome/free-solid-svg-icons";
 import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
 import { Box } from "@strapi/design-system/Box";
 import { Searchbar, SearchForm } from "@strapi/design-system/Searchbar";
@@ -12,7 +13,14 @@ import { CopyToClipboard } from "react-copy-to-clipboard";
 // https://www.thirdrocktechkno.com/blog/how-to-create-a-font-awesome-icons-plugin-for-strapi-cms/
 
 const Field = ({ name, value, onChange }) => {
-  const iconsArray = Object.keys(fab)
+  const [show, setShow] = useState(false)
+  const fabArray = Object.keys(fab).map(x => {
+    return `fab ${x}`
+  })
+  const fasArray = Object.keys(fas).map(x => {
+    return `fas ${x}`
+  })
+  const iconsArray = [].concat(fabArray,fasArray)
     .filter((icon) => {
       if (!value) {
         return true;
@@ -20,12 +28,20 @@ const Field = ({ name, value, onChange }) => {
       return icon.toLowerCase().includes(value.toLowerCase());
     })
     .sort();
-
+  const getIcon = (icon) => {
+    const iconName = icon.split(" ")[1] || ""
+    if (icon.includes('fab')){
+      return fab[iconName]
+    } 
+    if (icon.includes('fas')){
+      return fas[iconName]
+    }
+  }
   return (
     <>
       <Box>
         <Typography paddingBottom={9}>{name}</Typography>
-        <Box paddingBottom={8}>
+        <Box paddingBottom={1}>
           <SearchForm>
             <Searchbar
               name="searchbar"
@@ -39,6 +55,9 @@ const Field = ({ name, value, onChange }) => {
                 };
                 onChange(arg);
               }}
+              onFocus={() => {
+                setShow(true)
+              }}
               onClear={() => {
                 const arg = {
                   target: {
@@ -55,8 +74,9 @@ const Field = ({ name, value, onChange }) => {
             </Searchbar>
           </SearchForm>
         </Box>
+        {show &&
         <Box
-          padding={5}
+          padding={1}
           height="280px"
           overflow="scroll"
           background="neutral100"
@@ -79,12 +99,13 @@ const Field = ({ name, value, onChange }) => {
                         },
                       };
                       onChange(arg);
+                      setShow(false)
                     }}
                   >
                     <Tooltip description="Select Icon">
                       <CopyToClipboard text={icon}>
                         <Typography fontWeight="bold" as="button" type="button">
-                          <FontAwesomeIcon icon={fab[icon]} />
+                          <FontAwesomeIcon icon={getIcon(icon)} />
                         </Typography>
                       </CopyToClipboard>
                     </Tooltip>
@@ -93,7 +114,7 @@ const Field = ({ name, value, onChange }) => {
               );
             })}
           </Grid>
-        </Box>
+        </Box> }
       </Box>
     </>
   );
diff --git a/node_modules/strapi-plugin-icon-picker/admin/src/components/PluginIcon/index.js b/node_modules/strapi-plugin-icon-picker/admin/src/components/PluginIcon/index.js
old mode 100644
new mode 100755
diff --git a/node_modules/strapi-plugin-icon-picker/admin/src/index.js b/node_modules/strapi-plugin-icon-picker/admin/src/index.js
old mode 100644
new mode 100755
diff --git a/node_modules/strapi-plugin-icon-picker/admin/src/pages/App/index.js b/node_modules/strapi-plugin-icon-picker/admin/src/pages/App/index.js
old mode 100644
new mode 100755
diff --git a/node_modules/strapi-plugin-icon-picker/admin/src/pages/HomePage/index.js b/node_modules/strapi-plugin-icon-picker/admin/src/pages/HomePage/index.js
old mode 100644
new mode 100755
diff --git a/node_modules/strapi-plugin-icon-picker/admin/src/pluginId.js b/node_modules/strapi-plugin-icon-picker/admin/src/pluginId.js
old mode 100644
new mode 100755
diff --git a/node_modules/strapi-plugin-icon-picker/admin/src/translations/en.json b/node_modules/strapi-plugin-icon-picker/admin/src/translations/en.json
old mode 100644
new mode 100755
diff --git a/node_modules/strapi-plugin-icon-picker/admin/src/translations/fr.json b/node_modules/strapi-plugin-icon-picker/admin/src/translations/fr.json
old mode 100644
new mode 100755
diff --git a/node_modules/strapi-plugin-icon-picker/admin/src/utils/getTrad.js b/node_modules/strapi-plugin-icon-picker/admin/src/utils/getTrad.js
old mode 100644
new mode 100755
diff --git a/node_modules/strapi-plugin-icon-picker/server/bootstrap.js b/node_modules/strapi-plugin-icon-picker/server/bootstrap.js
old mode 100644
new mode 100755
diff --git a/node_modules/strapi-plugin-icon-picker/server/config/index.js b/node_modules/strapi-plugin-icon-picker/server/config/index.js
old mode 100644
new mode 100755
diff --git a/node_modules/strapi-plugin-icon-picker/server/content-types/index.js b/node_modules/strapi-plugin-icon-picker/server/content-types/index.js
old mode 100644
new mode 100755
diff --git a/node_modules/strapi-plugin-icon-picker/server/controllers/index.js b/node_modules/strapi-plugin-icon-picker/server/controllers/index.js
old mode 100644
new mode 100755
diff --git a/node_modules/strapi-plugin-icon-picker/server/controllers/my-controller.js b/node_modules/strapi-plugin-icon-picker/server/controllers/my-controller.js
old mode 100644
new mode 100755
diff --git a/node_modules/strapi-plugin-icon-picker/server/destroy.js b/node_modules/strapi-plugin-icon-picker/server/destroy.js
old mode 100644
new mode 100755
diff --git a/node_modules/strapi-plugin-icon-picker/server/index.js b/node_modules/strapi-plugin-icon-picker/server/index.js
old mode 100644
new mode 100755
diff --git a/node_modules/strapi-plugin-icon-picker/server/middlewares/index.js b/node_modules/strapi-plugin-icon-picker/server/middlewares/index.js
old mode 100644
new mode 100755
diff --git a/node_modules/strapi-plugin-icon-picker/server/policies/index.js b/node_modules/strapi-plugin-icon-picker/server/policies/index.js
old mode 100644
new mode 100755
diff --git a/node_modules/strapi-plugin-icon-picker/server/register.js b/node_modules/strapi-plugin-icon-picker/server/register.js
old mode 100644
new mode 100755
diff --git a/node_modules/strapi-plugin-icon-picker/server/routes/index.js b/node_modules/strapi-plugin-icon-picker/server/routes/index.js
old mode 100644
new mode 100755
diff --git a/node_modules/strapi-plugin-icon-picker/server/services/index.js b/node_modules/strapi-plugin-icon-picker/server/services/index.js
old mode 100644
new mode 100755
diff --git a/node_modules/strapi-plugin-icon-picker/server/services/my-service.js b/node_modules/strapi-plugin-icon-picker/server/services/my-service.js
old mode 100644
new mode 100755
diff --git a/node_modules/strapi-plugin-icon-picker/strapi-admin.js b/node_modules/strapi-plugin-icon-picker/strapi-admin.js
old mode 100644
new mode 100755
diff --git a/node_modules/strapi-plugin-icon-picker/strapi-server.js b/node_modules/strapi-plugin-icon-picker/strapi-server.js
old mode 100644
new mode 100755
